composer.json

```
@@ -5,15 +5,22 @@
     "license": "MIT",
     "autoload": {
         "psr-4": {
-            "Core\\SmarterU\\": "src/"
+            "CBS\\SmarterU\\": "src/"
         }
     },
     "authors": [
         {
             "name": "Brian Reich",
             "email": "brian.reich@thecoresolution.com"
+        },
+        {
+            "name": "Will Santanen",
+            "email": "will.santanen@thecoresolution.com"
         }
     ],
+    "require": {
+        "guzzlehttp/guzzle": "^7.0"
```

We also depend on the SimpleXML PHP extension
```suggestion
        "guzzlehttp/guzzle": "^7.0",
        "ext-SimpleXML": "*"
```